app/test_main.py

```diff
@@ -1,3 +1,50 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat,dog,expected",
+    [
+        (23, 23, [1, 1]),
+        (14, 14, [0, 0]),
+        (15.5, 15.1, [1, 1])
+    ]
+)
+def test_of_correct_counting(cat: int, dog: int, expected: list) -> None:
+    assert get_human_age(cat, dog) == expected
+
+
+@pytest.mark.parametrize(
+    "cat,dog,expected",
+    [
+        pytest.param(1000, 150, ValueError, id="extremely large value"),
```

Comment:
After you remove the changes from app/main.py, you can move that code to the `test_of_correct_counting `method.